integration_test.go

```diff
@@ -124,7 +145,23 @@ func TestPipeline(t *testing.T) {
 			t.Fatalf("Failed to read query results: %v", err)
 		}
 
+		//Verify the value of the data
```

Comment:
Is there a way to get the number of fields in the struct `log_entry_alltypes`? If so, the first check should be that number of fields in that struct is equal to the size of `BQvalues`. That can be a good indicator of what went wrong if it fails, and that will make sure that future changes will have a proper verification e2e.

Reply:
Done. I'm assuming this is unnecessary to add for the tests which only send one field, right?

Reply:
Yeah, no need in other tests. Only here when you have a huge struct and it's easy for things to get missed.